pipeline_dp/dp_engine.py

```diff
@@ -110,3 +114,20 @@ def unnest_cross_partition_bound_sampled_per_key(pid_pk_v):
         return self._ops.flat_map(col,
                                   unnest_cross_partition_bound_sampled_per_key,
                                   "Unnest")
+    
+    def _select_private_partitions(self, col):
+        """
+        Selects and publishes private partitions.
```

Comment:
Nit (=style improvement)
"""Selects private partitions  ( no need in publishes)